plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -1574,6 +1576,12 @@ public static function create_tables() {
 			}
 		}
 
+		if ( $wpdb->get_var( "SHOW TABLES LIKE '{$wpdb->prefix}wc_order_product_lookup';" ) ) {
+			if ( 2 > $wpdb->get_var( "SELECT COUNT(*) FROM INFORMATION_SCHEMA.STATISTICS WHERE TABLE_SCHEMA = DATABASE() AND TABLE_NAME = '{$wpdb->prefix}wc_order_product_lookup' AND INDEX_NAME = 'PRIMARY'" ) ) {
+				$result = $wpdb->query( "ALTER TABLE {$wpdb->prefix}wc_order_product_lookup DROP PRIMARY KEY, ADD PRIMARY KEY (order_item_id, order_id)" );
+			}
+		}
```

Comment:
Moved here so it can run before `$db_delta_result = dbDelta( self::get_schema() );`

Reply:
Thanks, makes sense! I removed it.

Reply:
Never mind, I just realised we are indeed checking the number of columns for the primary key